source/includes/code-examples/InsertOne.cs

```diff
@@ -0,0 +1,88 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CsharpExamples.UsageExamples;
+
+public class InsertOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        WriteLine("Inserting a document...");
+        InsertOneRestaurant();
+
+        // find and print newly inserted document
+        var document = _restaurantsCollection.Find(Builders<Restaurant>.Filter
+            .Eq(r => r.Name, "Mongo's Pizza")).FirstOrDefault();
+
+        WriteLine("Document Inserted: " + document.ToBsonDocument());
+
+        Cleanup();
+    }
+
+    private static void InsertOneRestaurant()
+    {
+        // delete sample document if already exists
+        Cleanup();
+
+        // start-insert-one
+        Restaurant newRestaurant = new()
+        {
+            Name = "Mongo's Pizza",
+            RestaurantId = "12345",
+            Cuisine = "Pizza",
+            Address = new BsonDocument
+            {
+                {"street", "Pizza St"},
+                {"zipcode", "10003"},
+            },
+            Borough = "Manhattan",
+        };
+
+        _restaurantsCollection.InsertOne(newRestaurant);
+        // end-insert-one
+    }
+
+    private static void Setup()
+    {
+        // This allows automapping of the camelCase database fields to our models. 
+        var camelCaseConvention = new ConventionPack { new CamelCaseElementNameConvention() };
+        ConventionRegistry.Register("CamelCase", camelCaseConvention, type => true);
+
+        // Establish the connection to MongoDB and get the restaurants database
+        var mongoClient = new MongoClient(_mongoConnectionString);
+        var restaurantsDatabase = mongoClient.GetDatabase("sample_restaurants");
+        _restaurantsCollection = restaurantsDatabase.GetCollection<Restaurant>("restaurants");
+    }
+
+    private static void Cleanup()
+    {
```

Comment:
I think it would be good to make sure that CleanUp has the same format everywhere (so move filter into a diff variable)

Reply:
same in some other places in this PR